README.md

```diff
@@ -806,6 +806,7 @@ It is sometime necessary to programmatically upload files to the server. Use the
                                prefix is the prefix of the actual filename used on the server (a timestamp is
                                added to ensure uniqueness)
     DELETE /data/<filename>  - Deletes the specified file (only if under control of the JobServer)
+    PUT /data?reset=reboot   - Deletes all uploaded files. Use ?sync=false to execute asynchronously.
```

Comment:
Hey @noorul, 
I only used PUT reset=reboot for consistency with existing functionality (PUT contexts?reset=reboot).
In fact, as I already pointed out [here](https://github.com/spark-jobserver/spark-jobserver/issues/888), PUT is not the correct method to use anyways.
I did not want to introduce inconsistency in the API, so I thought I would follow the current design and hopefully change it once the issue that I had opened would be approved.